### PR TITLE
Bugfix/search results export efficiency

### PR DESCRIPTION
**Pull request by nataliemona**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,8 +7,3 @@ Set the project up locally or in the cloud:
 
 Learn about the API endpoints: [API Documentation](api_documentation.md)
 
-Elasticsearch Setup: [Elasticsearch Documentation](elasticsearch_documentation.md)
```
Comment (nataliemona) [inline README.md:10]: Elasticsearch documentation is in developer manual now so this is a dead link

File: backend/ctgov/api/views.py [hunk 2]
```diff
@@ -242,11 +242,12 @@ def post(self, request, *args, **kwargs):
             .filter(q_study_roa)
             .filter(q_study_status)
             .filter(q_phase)
-            .all()
+            .all()[:100]
```
Comment (nataliemona) [inline backend/ctgov/api/views.py:245]: The fix

File: backend/ctgov/models.py [hunk 3]
```diff
@@ -1329,20 +1329,20 @@ class Endpoints(models.Model):
     study_title = models.TextField()
     study_type = models.TextField()
     entry_year = models.IntegerField()
-    study_id = models.CharField()
-    url = models.CharField()
-    indication = models.CharField()
-    treatment_name = models.CharField()
+    study_id = models.CharField(max_length=255)
```
Comment (nataliemona) [inline backend/ctgov/models.py:1332]: These were causing errors

File: backend/core/settings.py [hunk 4]
```diff
@@ -175,6 +175,11 @@
 # https://docs.djangoproject.com/en/3.2/releases/3.2/#customizing-type-of-auto-created-primary-keys
 DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
 
+if PROD:
```
Comment (nataliemona) [inline backend/core/settings.py:178]: Just moving the logs to a different place in production

**Review by swt144 — APPROVED** [review]

**Pull request merged**